### PR TITLE
[WIP][v8.0] Hackaton fixes 

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES

*Core
FIX: ES DB getDoc returns a DIRAC structure

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/Core/Utilities/ElasticSearchDB.py [hunk 1]
```diff
@@ -248,8 +251,6 @@ def getDoc(self, index: str, id: str) -> dict:
         except NotFoundError:
             sLog.debug("Document not found")
             return {}
-        except RequestError as re:
```
[the remark this answers is not in the document]
Comment (chaen) [reply src/DIRAC/Core/Utilities/ElasticSearchDB.py:251]: because it is caught and transformed to `S_ERROR` by `convertToReturnValue`

[the remark this answers is not in the document]
Comment (rupozzi) [reply src/DIRAC/Core/Utilities/ElasticSearchDB.py:251]: Ah ok good. Will `convertToReturnValue` make the method return `S_OK` in case no exceptions are caught?

[the remark this answers is not in the document]
Comment (chaen) [reply src/DIRAC/Core/Utilities/ElasticSearchDB.py:251]: yes

**Comment by chaen** [comment]
I'll just close that PR and let you handle it. There are just too many corner cases, and I don't have time to chase them all, you will be much faster than me. 
You really may want to add some more tests for the basic methods (like getDoc)
